examples/python_3dunet/visualization_utils.py

```diff
@@ -0,0 +1,126 @@
+import sys
```

Comment:
the following imports are not being used:
sys
os
pandas
re

Reply:
Can you please check this file? On github, I don't see it updated.

Reply:
oops, missed it, thanks, just updated it